scripts/utils/createJestConfig.js

```diff
@@ -76,8 +64,10 @@ module.exports = (resolve, rootDir, isEjecting) => {
     'collectCoverageFrom',
     'coverageReporters',
     'coverageThreshold',
+    'resetMocks',
+    'resetModules',
     'snapshotSerializers',
-    'moduleNameMapper',
```

Comment:
We've allowed `moduleNameMapper` a while ago (in 2.16.0) to be able to use the path mapping configurable for `typescript` in this case as well.

Reply:
Will moduleNameMapper be included?

Reply:
Just ran into this too Danny with my toy sample project!